src/lib/functions/netlify-function.js

```diff
@@ -76,7 +84,7 @@ class NetlifyFunction {
   async invoke(event, context) {
     await this.buildQueue
 
-    const timeout = this.isBackground ? this.timeoutBackground : this.timeoutSynchronous
+    const timeout = this.isBackground || (await this.isScheduled()) ? this.timeoutBackground : this.timeoutSynchronous
```

Comment:
Is this because we're triggering scheduled functions as background functions? We're not quite ready to do that yet, so I wonder if we should adjust the timeout then.

Reply:
addressed in 6e7f015. I thought about also changing the response behaviour of `handleScheduledFunction` to block the HTTP request (and thus, also the `ntl functions:invoke` call), but decided against it to better transport the concept of "kicking off" a cron job.